web/client/examples/viewer/plugins/mobile.jsx

```diff
@@ -105,6 +109,8 @@ module.exports = {
                             <LangBar key="langSelector"
                             currentLocale={props.locale}
                             onLanguageChange={props.loadLocale}/>
+                            <h5>Fullscreen</h5>
+                            <FullScreen fullscreen={false} propertiesToggle={props.toggleFullScreen} />
```

Comment:
If fullscreen is always false, the state of the button will never change. It should be tied to the current state